docs-src/pages/UiCalendar.vue

```diff
@@ -152,7 +154,7 @@
                                 <td></td>
```

Comment:
Can't comment on unchanged line above, but the `value` prop should not be `modelValue`.